src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1510,4 +1510,13 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="UseAssignableTypeForForeachVariableDescription" xml:space="preserve">
+    <value>There is no implicit conversion between collection element and iterator variable, this will likely result in runtime exception.</value>
```

Comment:
"will likely" seems stronger than it should be.

Reply:
Changed it to "can"